include/extractor/guidance/intersection_handler.hpp

```diff
@@ -57,6 +57,15 @@ class IntersectionHandler
     const IntersectionGenerator &intersection_generator;
     const NodeBasedGraphWalker graph_walker; // for skipping traffic signal, distances etc.
 
+    // Decide whether or not navigation should be suppressed on an edge based on mode type
+    bool SuppressModeNavigation(const TravelMode &in_mode,
+                                const TravelMode &out_mode,
+                                const SuppressModeListT &SUPPRESS_MODE_LIST) const;
+
+    bool SuppressModeNavigation(const EdgeID &via_edge,
```

Comment:
via_edge/connected are unintuitive to me. how about `from_edge` and `to_edge`? or `in_edge` and `out_edge`?